src/_gettsim/parameters/ges_rente.yaml

```diff
@@ -875,12 +875,12 @@ altersgrenze_für_frauen_vorzeitig:
 
 altersgrenze_langj_versicherte_vorzeitig:
   name:
-    de: Altersgrenze für vorzeitigen Bezug der Altersrente für langjährig Versicherte.
+    de: Altersgrenze für vorzeitigen Bezug der Altersrente für langjährig Versicherte
     en: Early retirement age for long term insured
   description:
     de: >-
-      Frühesmögliches Alter ab dem die Altersrente für langjährig Versicherte bezogen
-      werden kann (mit Abschlägen).
+      Frühesmögliches Alter ab dem die Altersrente für langjährig Versicherte be-
```

Comment:
Please don't break words. In general, why all those unrelated changes to the texts?

Reply:
I guess pre-commit hooks complaining about line length. I will ask Laura to run pre-commit hooks over all files to avoid it in the future